src/js/utils/helpers.ts

```diff
@@ -0,0 +1,41 @@
+import i18n from 'i18next';
```

Comment:
Maybe rename this file to `messageHelpers` or similar; it's oriented towards user-visible strings entirely.